lib/ReactViews/Map/Panels/HelpPanel/VideoGuide.jsx

```diff
@@ -0,0 +1,88 @@
+import { observer } from "mobx-react";
+import PropTypes from "prop-types";
+import React from "react";
+import { withTranslation } from "react-i18next";
+import { withTheme } from "styled-components";
+import styled from "styled-components";
+import Box from "../../../../Styled/Box";
+import FadeIn from "../../../Transitions/FadeIn/FadeIn";
+import Loader from "../../../Loader";
+
```

Comment:
I was briefly, slightly confused when I saw this and wondered where the videoguide from help panels were sourced from, but now I see you've joined the two 😆